### PR TITLE
[NFTR-20] Process list transactions

### PR DESCRIPTION
**Pull request by AdrianD97**
In this PR I've added the logic for processing ```list``` transactions.

### CODE REVIEW

File: src/processors/ScrsProcessor.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            } else {
+                console.warn(`Not enough arguments for list op. Expected 5, got ${tokens}`);
+            }
+            
+        }
+
+        const results: PromiseSettledResult<ListNftApiResponse>[] = await Promise.allSettled(promises);
+
+        results.forEach((result: PromiseSettledResult<ListNftApiResponse>) => {
+            if (result.status === "rejected") {
```
Comment (AdrianD97) [inline src/processors/ScrsProcessor.ts:93]: @trascaandrei  what about saving to a queue these rejected promises and try later again?
Why could these promises be rejected? Maybe the nftr BE is down or a network problem occurred.
I am not sure what will be the impact if a list transaction is not stored into nftr DB.

Comment (trascaandrei) [reply src/processors/ScrsProcessor.ts:93]: Yes, definitely. The impact is the fact that those NFT won't be available on the website and it is exactly the problem that we are trying to solve with this observer

**Review by trascaandrei — APPROVED** [review]

File: src/processors/ScrsProcessor.ts (hunk 1 above)
Comment (AdrianD97) [reply src/processors/ScrsProcessor.ts:93]: I will add a task on Jira for implementing a retry mechanism for these rejected requests.

Comment (AdrianD97) [reply src/processors/ScrsProcessor.ts:93]: Jira task: https://trsdesignro.atlassian.net/browse/NFTR-29

**Pull request merged**